css/footer.css

```diff
@@ -0,0 +1,40 @@
+/* FOOTER STYLING */
+.footer-border-line {
+    border-top: 2px solid #dfdfdf;
+    margin: 0 150px 0 150px;
+    width: 100%;
+    margin-left: auto;
+    background-color: rgb(244, 114,208);
+    margin-right: auto;
+
+} 
+
+.footer h1 {
```

Comment:
It makes me confused as footer  and h1 are element names in HTML, may be it would be better rename for this class.